library/src/main/java/com/mvc/imagepicker/ImagePicker.java

```diff
@@ -228,6 +316,66 @@ public static Bitmap getImageFromResult(Context context, int requestCode, int re
 
     /**
      * Called after launching the picker with the same values of Activity.getImageFromResult
+     * in order to resolve the result and get the image path.
+     *
+     * @param context             context.
+     * @param requestCode         used to identify the pick image action.
+     * @param resultCode          -1 means the result is OK.
+     * @param imageReturnedIntent returned intent where is the image data.
+     * @return path to the saved image.
+     */
+    @Nullable
+    public static String getImagePathFromResult(Context context, int requestCode, int resultCode,
```

Comment:
Instead of getting decoded bitmap it's also usefull to get just a file path. In this case you could temporary save the path to load later or make simple load with e.g. Picasso lib.